misc-utils/meson.build

```diff
@@ -0,0 +1,22 @@
+uuidd_service = configure_file(
+  input : 'uuidd.service.in',
+  output : 'uuidd.service',
+  configuration : conf)
+
+uuidd_socket = configure_file(
+  input : 'uuidd.socket.in',
+  output : 'uuidd.socket',
+  configuration : conf)
+
+install_data(uuidd_service,
+             install_dir : systemdsystemunitdir)
+install_data(uuidd_socket,
+             install_dir : systemdsystemunitdir)
+
+if get_option('enable-uuidd')
+  uuidd_8 = configure_file(
+    input : 'uuidd.8.in',
+    output : 'uuidd.8',
+    configuration : conf)
+  install_man(uuidd_8)
+endif
```

Comment:
getopt.1 is generated file, seems we need:

getopt_1 = configure_file(
  input : 'getopt.1.in',
  output : 'getopt.1',
  configuration : conf)
install_man(getopt_1)